spyder/utils/icon_manager.py

```diff
@@ -209,15 +209,15 @@
     '2downarrow':              [('fa.angle-double-down',), {'color': MAIN_FG_COLOR}],
     '1downarrow':              [('fa.angle-down',), {'color': MAIN_FG_COLOR}],
     # --- Autocompletion type icons --------------
-    'attribute':               [('spyder.circle-letter-a',), {'color': 'magenta'}],
-    'module':                  [('spyder.circle-letter-m',), {'color': '#daa520'}],
-    'class':                   [('spyder.circle-letter-c',), {'color':'#3775a9'}],
-    'private2':                [('spyder.circle-underscore',), {'color':'#e69c9c'}],
-    'private1':                [('spyder.circle-underscore',), {'color':'#e69c9c'}],
-    'method':                  [('spyder.circle-letter-m',), {'color':'#7ea67e'}],
-    'function':                [('spyder.circle-letter-f',), {'color':'orange'}],
-    'blockcomment':            [('spyder.circle-hash',), {'color':'grey'}],
-    'cell':                    [('spyder.circle-percent',), {'color':'red'}],
+    'attribute':               [('mdi.alpha-a-box',), {'color': 'magenta','scale_factor': 1.3}],
```

Comment:
Add a space between ``,`` and ``'scale_factor`` on these, per PEP8. I honestly wouldn't worry too overly much about the line length being a little long in this specific case, to say consistent with the other icons all being on one line and since this is not a main code file.